src/l-image-overlay.js

```diff
@@ -38,6 +38,10 @@ class LImageOverlay extends LLayer {
     );
   }
 
+  disconnectedCallback() {
+    this.layer.remove();
```

Comment:
Pretty sure this can't be `null` but Typescript will want a null check here :-)

Reply:
okidokes!